blibli-backend-framework-newrelic/src/main/java/com/blibli/oss/backend/newrelic/newrelic/configuration/NewRelicReactiveMongoInstrumentationAutoConfiguration.java

```diff
@@ -0,0 +1,30 @@
+package com.blibli.oss.backend.newrelic.newrelic.configuration;
+
+import com.blibli.oss.backend.newrelic.newrelic.aspect.CommandAspect;
+import com.blibli.oss.backend.newrelic.newrelic.aspect.ReactiveMongoDbAspect;
+import com.blibli.oss.backend.newrelic.newrelic.aspect.service.AspectModifyService;
+import com.blibli.oss.backend.newrelic.newrelic.aspect.service.impl.AspectModifyServiceImpl;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.boot.autoconfigure.mongo.MongoProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import org.springframework.data.mongodb.core.ReactiveMongoTemplate;
+
+import java.util.Optional;
+
+@Configuration
+@ConditionalOnClass({EnableNewRelicReactorInstrumentation.class})
+@Slf4j
+@Import({CommandAspect.class, ReactiveMongoDbAspect.class})
+public class NewRelicReactiveMongoInstrumentationAutoConfiguration {
+
+  @Bean
+  public AspectModifyService aspectHelper(
+    Optional<MongoProperties> mongoProperties,
```

Comment:
Is this already working for spring project without mongodb? This looks like unchanged from previous version.

Reply:
the service is already separated, it should be good now